tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
```

Comment:
I copied these fixtures from the rdmo repo. We can of course pull this file from the rdmo during ci.
But I guess we need to have additional fixtures here.